src/main/resources/swagger/api-docs.yaml

```diff
@@ -2845,22 +2821,6 @@ definitions:
       curator:
         type: boolean
 
-  DeleteEntities:
```

Comment:
This structure is not needed because we don't allow the user to make recursion decisions
* deletions causing dangling references are prohibited
* we do not delete any entities that aren't supplied directly